proof/invariant-abstract/DetSchedInvs_AI.thy

```diff
@@ -948,9 +997,6 @@ lemmas vs_pred_map_simps =
   pred_map_def[where m="kheap s" for s :: "'z state"]
 
 lemmas vs_heap_simps =
-  tcb_of_simps
```

Comment:
It's fine to leave them removed here if they are declared `[simp]` at the point we generate them with `[split_simps ..]`